### PR TITLE
API Doc

### PR DESCRIPTION
**Pull request by sungbae**
This is not finished yet. I just want it out as a review for now. I want to go over the API with @untra.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,3 +77,175 @@ wget -qO- http://download.rethinkdb.com/apt/pubkey.gpg | sudo apt-key add -
...
+    "departments": ["Computer Science"],
+    "major4": "",
+    "major1": "",
+    "username": "suba8204",
+    "primary_affiliation": ["Student"],
+    "minor2": "",
+    "registration": "registration_culdap"
+ }
+
+ Note: It did not update my information, instead it just gave back my old information.
```
Comment (sungbae) [inline README.md:181]: Look here

File: README.md [hunk 2]
```diff
@@ -77,3 +77,175 @@ wget -qO- http://download.rethinkdb.com/apt/pubkey.gpg | sudo apt-key add -
...
+*Test: FAIL*
+```
+GET /api/surveys
+```
+**Response:**
+```
+Status: 200 OK
+[]
+
+Note: I need to test this out with an actual survey that is stored and tied with the User/Question tables.
```
Comment (sungbae) [inline README.md]: Look here.

File: README.md [hunk 3]
```diff
@@ -77,3 +77,175 @@ wget -qO- http://download.rethinkdb.com/apt/pubkey.gpg | sudo apt-key add -
...
+| title            | String        | **Required**. This is the actual question itself (i.e. Did you like this course?).         |
+| response_format  | String        | **Required**. Types of responses are: trueOrFalse, multipleChoice, freeResponse, etc.      |
+| options          | String[]      | **Required**. If it is a multipleChoice question, you must provide the options.            |
+
+**Response:**
+```
+Status: 200 OK
+null
+
+NOTE: Bug? This needs to tie in with survey and user tables. All it does right now is just create/update question tables.
```
Comment (sungbae) [inline README.md]: Look here.

edit: Nevermind, it works fine for Course and User.

**Comment by sungbae** [comment]
Does someone want to look over this and merge it? I think it is ready.

**Comment by untra** [comment]
@sungbae I'll look at it
Super simple changes. Personally I think the API documentation should live in it's own wiki page, but we can deal with that later (it also would not be a pr)

**Pull request merged**